cmd/crio/main.go

```diff
@@ -362,9 +362,10 @@ func main() {
 			Name:  "storage-opt",
 			Usage: fmt.Sprintf("storage driver option (default: %q)", defConf.StorageOptions),
 		},
+		// XXX: DEPRECATED
 		cli.BoolFlag{
 			Name:  "file-locking",
-			Usage: fmt.Sprintf("enable or disable file-based locking (default: %t)", defConf.FileLocking),
+			Usage: fmt.Sprintf("enable or disable file-based locking (depreciated) (default: %t)", defConf.FileLocking),
```

Comment:
Do we want to use `deprecated` instead of `depreciated`?

Reply:
Should we just Hide the option.  I see no reason to continue to document an option that we no longer want to support.  We can put a reference into the man page, but no reason to show the option crio --help

Reply:
Any concerns about people already using the option and adjustments they might need to make?

Reply:
I am personally not concerned about it. I don't think it's thought about very much

Reply:
Existing users will continue to work, if hidden, and document the deprecation in the man page.